src/confluent_kafka/schema_registry/schema_registry_client.py

```diff
@@ -62,7 +62,26 @@ def _urlencode(value: str) -> str:
 VALID_AUTH_PROVIDERS = ['URL', 'USER_INFO']
 
 
-class _OAuthClient:
+class _BearerFieldProvider:
+    def get_bearer_fields(self) -> dict:
+        raise NotImplementedError
+
+
+class _StaticFieldProvider(_BearerFieldProvider):
```

Comment:
Add `__init__` that takes a token and sets a local `bearer_token` member variable.  Then you can remove the `bearer_token` member variable in the SR client.